machines/vcItem.ts

```diff
@@ -407,7 +407,11 @@ export const vcItemMachine =
               entry: 'removeVcItem',
               on: {
                 STORE_RESPONSE: {
-                  actions: ['removedVc', log('removing Vc')],
+                  actions: [
+                    'removedVc',
+                    log('removing Vc'),
```

Comment:
If this xstate's `log` is equivalent to `console.log` and is only for debugging purposes, it can be removed once the changes are tested.

Reply:
It is just for testing purpose, we can remove the log